create-release

```diff
@@ -158,7 +160,7 @@ else
           fi
         fi
       fi
-      if ! echo "$version" | grep -q '^[0-9]\+[0-9a-z.]*\([\~-][0-9a-z.\~-]*\)\?$' ; then
+      if ! echo "$version" | grep -q '^\([0-9]\+:\)\?[0-9]\+[0-9a-z.]*\([\~-][0-9a-z.\~-]*\)\?$' ; then
```

Comment:
Consider using `/bin/echo` instead as built-in `echo` might fail on [Broken pipe error](https://www.unix.com/unix-for-dummies-questions-and-answers/149108-echo-write-error-broken-pipe.html).

Reply:
hmm, good point, but with grep this should not happen.. many other scrips do this like this as well..

Reply:
Unfortunately, we experienced this issue in Jenkins pipeline even for grep.